app/src/main/java/com/zulip/android/activities/LoginActivity.java

```diff
@@ -96,6 +109,12 @@ public void onClick(View view) {
     private void showLoginFields() {
         AnimationHelper.showView(findViewById(R.id.serverInput), 201);
         AnimationHelper.hideView(findViewById(R.id.serverFieldLayout), 100);
+        realmNameET = (EditText) findViewById(R.id.realmName);
+        if (getIntent().getBooleanExtra("FROM_ADDREALM", false)) {
+            startedFromAddRealm = true;
+            ((TextView) findViewById(R.id.textView1)).setText(R.string.add_realm);
```

Comment:
@kunall17 this is possibly a rebasing error, but i can't get my version of this to compile because it can't find the `textView1` view.